presto-product-tests/conf/docker/multinode/compose.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+source ${BASH_SOURCE%/*}/../../../bin/locations.sh
```

Comment:
it resolves the 
PRESTO_SERVER_DIR
PRESTO_CLI_JAR
PRODUCT_TESTS_JAR
properties (or rahter: their default values). Docker-compose does not have support for default parameter values.